collector/src/cgroups.h

```diff
@@ -53,6 +57,118 @@ std::string CGroupDetected2string(CGroupDetected k);
 // Types
 //------------------------------------------------------------------------------
 
+#ifdef PROMETHEUS_SUPPORT
+/* structure for prometheus output : CPU utilization as reported by cpuacct cgroup */
+static const prometheus_kpi_descriptor prometheus_kpi_cgroup_cpu[] = {
+    // cgroup : cpu
+    { "cgroup_cpuacct_stats_user", KPI_TYPE::Gauge, "CPU time consumed by tasks in user mode" },
```

Comment:
I think 99% of these Kpis are KPI_TYPE::Counter since they are monotonically increasing...